spec/models/article_spec.rb

```diff
@@ -401,35 +405,40 @@ def build_and_validate_article(*args)
 
   describe "#async_score_calc" do
     context "when published" do
-      before { ActiveJob::Base.queue_adapter = :inline }
-
       let(:article) { build(:article) }
 
       it "updates the hotness score" do
-        article.save
-        article.reload
-        expect(article.hotness_score > 0).to eq(true)
+        perform_enqueued_jobs do
+          article.save
+          expect(article.reload.hotness_score.positive?).to eq(true)
```

Comment:
I noticed that `expect` is sometimes inside the `perform_enqueued_jobs` and sometimes outside of it. Is there a specific reason for it? Shouldn't it be outside of the block?

Reply:
Keen eye, it's a question of scoping mixed with laziness. Some expects require variables that are defined inside the block, some don't. I didn't pay too much attention to it but I can improve the code if you want

Reply:
I see. I don't think it's a critical thing to fix.

Reply:
@lightalloy I made the change in the meantime but I saw you already approved this PR and I don't want to have you redo the review, sorry 😅

Reply:
@rhymes that's ok, push your changes if you want them to be in this pr.

Reply:
@lightalloy I ended up reverting the changes, I suspect there's something going on with having two separate job queues but for now it will do. It'll probably fix itself when the transition to AJ is complete. Thanks again!